hnix-store-core/src/System/Nix/ReadonlyStore.hs

```diff
@@ -10,19 +10,20 @@ import qualified Data.HashSet as HS
 import           Data.Text (Text)
 import qualified Data.Text as T
 import           Data.Text.Encoding
-import           System.Nix.Internal.Hash
+import           System.Nix.Hash
 import           System.Nix.Path
 
 makeStorePath :: Text -> Text -> Digest 'SHA256 -> Text -> Path
```

Comment:
Were you going to rename `Path` to `StorePath`?

Reply:
Yeah, I just scoped this original PR down to just the Hash module. After this is in I'll merge the ready open PRs then continue cleanup module by module